### ISSUE TITLE
ModuleNotFoundError: No module named 'prometheus_client'

### ISSUE THREAD

**Issue by grimsi** [LOENS2/pterodactyl_exporter#4]
Looks like prometheus_client is missing in your requirements.txt
Was able to fix the bug by running `pip3 install prometheus_client`

Logs:
```
Traceback (most recent call last):
  File "/usr/local/bin/pterodactyl_exporter", line 5, in <module>
    from pterodactyl_exporter.pterodactyl_exporter import main
  File "/usr/local/lib/python3.10/site-packages/pterodactyl_exporter/pterodactyl_exporter.py", line 6, in <module>
    from pterodactyl_exporter import config_load, http_client, http_server
  File "/usr/local/lib/python3.10/site-packages/pterodactyl_exporter/http_server.py", line 1, in <module>
    from prometheus_client import Gauge, start_http_server
ModuleNotFoundError: No module named 'prometheus_client'
```

**Comment by LOENS2** [comment]
You're absolutely correct. I don't know how that could happen, because my IDE usually adds dependencies to the requirements.txt by itself. I'm going to fix that very soon.

**Comment by LOENS2** [comment]
As it turns out, my IDE did a terrible job at managing the requirements.txt. I fixed it manually and the pip package will be updated soon.

**Comment by LOENS2** [comment]
The reason why I didn't notice the missing module while testing was, because I still had it installed from previous tests and forgot to reset my testing vm.

**Comment by LOENS2** [comment]
Resolved in `c6cb11f`
